apps/astarte_appengine_api/mix.exs

```diff
@@ -69,7 +69,7 @@ defmodule Astarte.AppEngine.API.Mixfile do
 
   defp astarte_required_modules(_) do
     [
-      {:astarte_core, github: "astarte-platform/astarte_core"},
+      {:astarte_core, github: "eddbbt/astarte_core", branch: "keyspace_feature", override: true},
```

Comment:
Now that the PR in `astarte_core` has been merged, you can point again to the main repo (here and in the other `mix.exs`s)